### ISSUE TITLE
SCSS partials imported after _settings not overriding previously imported properties

### ISSUE THREAD

**Issue by probablyabear** [foundation/foundation-emails#831]
I've noticed that SCSS properties I define in a SCSS partial that I import _after_ the _settings.scss file do not override properties defined in _settings.scss.

[Per Zurb's documentation](https://zurb.com/university/lessons/wrangling-sass-import-files): 

> Sass is sensitive to the order of selectors. That is, later commands in .scss files override earlier commands. Therefore you should always import your partials after Foundation to make the site your own.

Shouldn't the properties in my partial that's the last item in the `@import` list in app.scss override any properties contained in partials that were imported earlier in the chain?

**Comment by probablyabear** [comment]
After further testing, I found that this issue was not caused by Foundation. Rather, it seems to be due to CSS specificity. 

I found that I was able to make my property values in my custom partial take effect if I included the element's prior class. 

As an example, let's say I'm trying to change the background color of a `<container>` using a class I've defined as `body-container`. The HTML generated upon saving the project generates:  
`<table class="container body-container float-container></table>`

If I define my class a simply `.body-container`, the styles I define within it do not get applied to the table. However, if I define the class as `.container.body-container`, the styles do get applied to the table.

Is this difference inherent in CSS, or is it due to the way Foundation compiles all the styles?
